Backpack/Calendar/Classes/BPKCalendarYearPill.m

```diff
@@ -0,0 +1,55 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKCalendarYearPill.h"
+
+#import <Backpack/Color.h>
+#import <Backpack/Font.h>
+#import <Backpack/Spacing.h>
+
+@implementation BPKCalendarYearPill
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        self.translatesAutoresizingMaskIntoConstraints = NO;
+        self.attributedText = [BPKFont attributedStringWithFontStyle:BPKFontStyleTextXsEmphasized content:@"" textColor:[BPKColor white]];
```

Comment:
This line is redundant since we always set the year before displaying

Reply:
👍